src/index.js

```diff
@@ -57,6 +66,34 @@ class MongoSlowQueryChecker {
    * ```
    */
   async get() {
+    const ops = this.useProfiling
+      ? await this.getFromSystemProfile()
+      : await this.getCurrentlyRunningQueries();
+
+    return this.processOps(ops);
+  }
+
+  /**
+   * Get relevant (slow/COLLSCAN) queries from the `system.profile` table, ensuring we don't
+   * repeatedly return the same queries in consecutive runs.
+   */
+  async getFromSystemProfile() {
+    this.profile = this.profile || (await this.db.connect()).collection('system.profile');
+
+    const conditions = [{ millis: { $gte: this.queryThreshold * 1000 } }];
+    if (this.reportAllCollscans) conditions.push({ planSummary: { $regex: /COLLSCAN/ } });
```

Comment:
Are `COLLSCAN`s included in `system.profile` even if they don't exceed the servers slow query threshold?

Reply:
No. You have to use a zero threshold to get this. I'll document this requirement.

Reply:
Done.